topic_monitor/setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup
+
+setup(
+    name='topic_monitor',
+    version='0.0.0',
+    packages=[],
+    py_modules=[
+        'launch_files.launch_depth_demo',
+        'launch_files.launch_durability_demo',
+        'launch_files.launch_fragmentation_demo',
+        'launch_files.launch_reliability_demo',
+        'scripts.data_publisher',
+        'scripts.topic_monitor',
```

Comment:
The console scripts look good. But a package should only install code under its own namespace. This package does currently install `scripts` which could collide arbitrarily with any other package.